garage/np/algos/cem.py

```diff
@@ -50,6 +50,7 @@ def __init__(self,
                  extra_std=1.,
                  extra_decay_time=100,
                  **kwargs):
+        self.quick_init(locals())
```

Comment:
Please leave a comment listing the fields that cannot be pickled without this.

Reply:
i think he just missed adding the pickling interface (and removing Serializable) on numpy-based algos

Reply:
Serializable is removed from np algos now.